app/lib/slack_helper.rb

```
@@ -0,0 +1,134 @@
+class SlackHelper
+  def self.get_channel_info(channel_name)
+    client = Slack::Web::Client.new
+
+    begin
+      @response = client.channels_info(channel: '#' + channel_name)
+      return @response.channel.id
+    rescue Slack::Web::Api::Errors::SlackError => e
+      return nil
+    end
+  end
+
+  def self.post_user_update_to_channel(user, checkin, qotd_answer)
```

It seems that this should return an error message so when you're calling it in the controller and the `if` fails then you can render a why.